tests/client/coll.rs

```diff
@@ -7,13 +8,17 @@ use mongodb::client::MongoClient;
 fn find_and_insert() {
     let client = MongoClient::with_uri("mongodb://localhost:27017").unwrap();
     let db = client.db("test");
-    let coll = db.collection("test");
+    let coll = db.collection("find_and_insert");
 
     db.drop_database().ok().expect("Failed to drop database");
 
     // Insert document
-    let mut doc = bson::Document::new();
-    doc.insert("title".to_owned(), Bson::String("Jaws".to_owned()));
+    let doc = doc! {
+        "title" => Bson::String("Jaws".to_owned())
+    };
+
+    // let mut doc = bson::Document::new();
```

Comment:
Another good rule to follow: don't leave dead code hanging around in comments. Once code is commented out, it's difficult for people later on to figure out whether its safe to remove, so it often just lives on in the code base forever. If you want to find what the state of the code was previously, that's what the git history is for.